portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationWebBehavior.testcase

```diff
@@ -1373,4 +1373,46 @@ definition {
 		ACUtils.viewNameList(nameList = "ac ac");
 	}
 
+	@description = "Bug: LRAC-8233 | Automation ID: LRAC-11537 | Test Summary: Check web behavior shows which data source data came from"
+	@priority = "3"
+	test CheckWebBehaviorSegmentShowsWhichDataSourceDataCameFrom {
+		ACUtils.createBlogsAndAddToPage();
+
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
```

Comment:
Hey @calvinctvo, you don't need to put a variable to receive the macro's return, this macro will create a global variable for all tests that use it with the name of the property. To access the property name you just need to use like this:

`${assignedPropertyName}`